### PR TITLE
Centered spinner in viewheight

### PR DESCRIPTION
**Pull request by iDavidB**
It bothered me a lot that the spinner was always on the top left, especially on the homepage where it doesn't show any breadcrumbs while loading.

### CODE REVIEW

File: src/component/Spinner.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+export default function Spinner() {
+  const classes = useStyles();
+
+  return (
+    <div className={classes.spinner}>
+      <CircularProgress />
+    </div>
+  );
+}
```
Comment (lucacasonato) [inline src/component/Spinner.tsx:21]: Can't you just center using flexbox? `style={{display: "flex", justifyContent: "center"}}` on the wrapper `div`. This adds a lot of JS processing which really isn't needed.

Comment (iDavidB) [reply src/component/Spinner.tsx:21]: I used flex box initially but noticed the spinner still being off-center when something else is rendered (The breadcrumb). This being absolutely positioned will make sure it's always perfectly centered. I could apply the styling in-line if that is preferred though.

**Comment by ry** [comment]
Thanks for the patch! I'll wait for @lucacasonato's approval.

File: src/component/Spinner.tsx (hunk 1 above)
Comment (lucacasonato) [reply src/component/Spinner.tsx:21]: Ah, makes sense then. :+1:

**Pull request merged**